src/redux/books/books.js

```diff
@@ -0,0 +1,29 @@
+const ADD_BOOK = 'bookstore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
+
+const initialState = [];
+
+export const addBook = (payload) => ({
+  type: ADD_BOOK,
+  payload,
+});
+
+export const removeBook = (id) => ({
+  type: REMOVE_BOOK,
+  id,
+});
+
```

Comment:
- [ ] Same goes for the remove button. Nothing happens when the button is clicked. The expected behavior is to dispatch the `removeBook()` action and remove the book from your store.